docs/src/pages/_app.tsx

```diff
@@ -22,7 +23,7 @@ function MyApp({ Component, pageProps }: AppProps) {
     );
   }
   return (
-    <ThemeProvider theme={appTheme}>
+    <ThemeProvider overrideTheme={ThemesEnum.LIGHT}>
```

Comment:
why we need this `overrideTheme` prop? removing it doesn't change anything

Reply:
For landing page
Approach with CSS media queries affects to landing page as well

Reply:
I don't see a difference? let's discuss it